contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -1366,6 +1366,8 @@ exec_stmt_decl_table(PLtsql_execstate *estate, PLtsql_stmt_decl_table *stmt)
 		}
 
 		tblname = psprintf("%s_%d", var->refname, estate->nestlevel);
+		if (strlen(tblname) > 63)
```

Comment:
is it possible for collision here with multiple table variables longer than 63 chars?

Reply:
Hmm, it might be possible. Let me test it out with multiple calls within a single transaction and see what happens.

Reply:
Confirmed that there is a crash when nesting TVFs which use the same table name. Will fix by keeping the nest level during the truncation.

Reply:
Also added a test for the nested function scenario.

Reply:
let's use NAMEDATALEN here as well. it seems like the other places in the code checking for identifier length do a comparison 

`if (len >= NAMEDATALEN)` so we should match that behavior